graphdb/janus/src/main/java/org/apache/atlas/repository/graphdb/janus/AtlasElasticsearchQuery.java

```diff
@@ -336,5 +341,14 @@ public double getScore() {
             }
             return Double.parseDouble(String.valueOf(hit.get("_score")));
         }
+
+        @Override
+        public Map<String, List<String>> getHighLights() {
+            Object highlight = this.hit.get("highlight");
+            if(Objects.nonNull(highlight)) {
+                return (Map<String, List<String>>) highlight;
```

Comment:
@arpit-at  I think I missed this yesterday, will there be a possibility of casting exception at L#349 ?

Reply:
It should be fine - I have checked it in scenarios of empty and non-empty highlight